map_server/CMakeLists.txt

```diff
@@ -94,6 +94,7 @@ if(BUILD_TESTING)
   )
 endif()
 
+ament_export_libraries(${PROJECT_NAME})
```

Comment:
`${PROJECT_NAME}` is an executable (https://github.com/ros2/navigation/pull/7/files#diff-17895ab0833c9808dd833590ac8871beR39)?